docs/installation.rst

```diff
@@ -234,6 +234,22 @@ dependencies.
     cd ..
     OPENSSL_DIR="${CWD}/openssl" pip wheel --no-cache-dir --no-binary cryptography cryptography
 
+Building a wheel with the Limited API
```

Comment:
Yeah, in the same vein, we shouldn't need to document this because it should be the default :-)